OpenEdXMobile/src/test/java/org/edx/mobile/test/ConfigTests.java

```diff
@@ -45,8 +45,8 @@
     private static final String TYPE = "TYPE";
     private static final String DETAIL_TEMPLATE = "DETAIL_TEMPLATE";
     private static final String FACEBOOK_APP_ID = "FACEBOOK_APP_ID";
-    private static final String FABRIC_KEY = "FABRIC_KEY";
-    private static final String FABRIC_BUILD_SECRET = "FABRIC_BUILD_SECRET";
+    private static final String KEY = "KEY";
+    private static final String SECRET = "SECRET";
     private static final String KITS = "KITS";
     private static final String CRASHLYTICS = "CRASHLYTICS";
     private static final String ANSWERS = "ANSWERS";
```

Comment:
Please remove *KITS*, *CRASHLYTICS*, *ANSWER* as well

Reply:
Plus If you can also remove these keys as well *PARSE*, *PARSE_ENABLED*, *PARSE_APPLICATION_ID*, *PARSE_CLIENT_KEY*